broker/src/main/java/pt/ulisboa/tecnico/softeng/broker/domain/ServiceLayer.java

```diff
@@ -0,0 +1,69 @@
+package pt.ulisboa.tecnico.softeng.broker.domain;
+
+import pt.ulisboa.tecnico.softeng.broker.services.remote.*;
+
+public class ServiceLayer {
+  private ActivityInterface activityInterface;
+  private HotelInterface hotelInterface;
+  private CarInterface carInterface;
+  private BankInterface bankInterface;
+  private TaxInterface taxInterface;
+
+  public ServiceLayer() {
```

Comment:
can this be `protected` ?

Reply:
Seemingly it can not. Because we need to create a `ServiceLayer` in `BrokerInterface` that is located in another package. 

However if we apply the builder pattern (recommended in #3) we can make the constructor `private`.